systemtests/src/main/java/io/enmasse/systemtest/utils/IoTUtils.java

```diff
@@ -4,35 +4,60 @@
  */
 package io.enmasse.systemtest.utils;
 
+import io.enmasse.address.model.Address;
 import io.enmasse.address.model.AddressSpace;
+import io.enmasse.admin.model.v1.AddressSpacePlan;
+import io.enmasse.iot.model.v1.AdapterConfig;
+import io.enmasse.iot.model.v1.AdaptersConfig;
 import io.enmasse.iot.model.v1.IoTConfig;
 import io.enmasse.iot.model.v1.IoTProject;
 import io.enmasse.iot.model.v1.IoTProjectBuilder;
+import io.enmasse.systemtest.Endpoint;
+import io.enmasse.systemtest.amqp.AmqpClient;
+import io.enmasse.systemtest.iot.HttpAdapterClient;
+import io.enmasse.systemtest.iot.MessageSendTester;
+import io.enmasse.systemtest.iot.MessageType;
 import io.enmasse.systemtest.logs.CustomLogger;
 import io.enmasse.systemtest.model.addressplan.DestinationPlan;
-import io.enmasse.systemtest.model.addressspace.AddressSpacePlans;
 import io.enmasse.systemtest.platform.Kubernetes;
 import io.enmasse.systemtest.time.SystemtestsOperation;
 import io.enmasse.systemtest.time.TimeMeasuringSystem;
 import io.enmasse.systemtest.time.TimeoutBudget;
+import io.enmasse.systemtest.time.WaitPhase;
+import io.fabric8.kubernetes.api.model.apps.Deployment;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.web.client.HttpResponse;
 import org.slf4j.Logger;
 
+import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Collections;
+import java.util.List;
 import java.util.Map;
 import java.util.Objects;
+import java.util.Optional;
 import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+import java.util.function.Function;
+
+import static io.enmasse.systemtest.apiclients.Predicates.any;
+import static java.net.HttpURLConnection.HTTP_ACCEPTED;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.fail;
 
 public class IoTUtils {
 
-    private static final String[] EXPECTED_DEPLOYMENTS = new String[]{
+    private static Kubernetes kubernetes = Kubernetes.getInstance();
+
+    private static final String IOT_SIGFOX_ADAPTER = "iot-sigfox-adapter";
```

Comment:
I think it is ok to drop the constants here and confine everything in the method `getExpectedDeploymentsNames`. This would put it all in one place, rather than having a mix String, String[] constants and the actual logic.

Reply:
In what way was this resolved?